test/Dialect/SMT/bitvector-errors.mlir

```diff
@@ -89,8 +96,8 @@ func.func @repeat_count_too_large(%arg0: !smt.bv<32>) {
 
 // -----
 
-func.func @repeat_result_type_bitwidth_too_large(%arg0: !smt.bv<18446744073709551612>) {
-  // expected-error @below {{result bit-width (provided integer times bit-width of the input type) must fit into 64 bits}}
-  smt.bv.repeat 2 times %arg0 : !smt.bv<18446744073709551612>
```

Comment:
In the repeat parser, 64 is hardcoded as the width of the unsigned integer used. We'd need to adjust this and keep this error test but with a smaller number.

Reply:
Cool, I've updated that now.